flow/core/kernel/scenario/aimsun.py

```diff
@@ -50,7 +50,12 @@ def generate_network(self, scenario):
             "osm_path": scenario.net_params.osm_path,
             'render': self.sim_params.render,
             "sim_step": self.sim_params.sim_step,
-            "traffic_lights": None
+            "traffic_lights": None,
+            "scenario_name": self.sim_params.scenario_name,
```

Comment:
how is this different from `scenario.name` or `scenario.orig_name`?

Reply:
`scenario_name` is the name of the scenario used by Aimsun, eg `Dynamic Scenario 917`. When creating a template we would set this equal to `scenario.name`, but when loading one, it can have several scenarios one of which we need to choose.